client/src/js/components/bhInventorySelect.js

```diff
@@ -24,9 +24,11 @@ function InventorySelectController(Inventory, Notify) {
   $ctrl.$onInit = function onInit() {
     // fired when an inventory has been selected
 
-    const params = $ctrl.onlyConsumable
-      ? { consumable : 1 }
-      : {};
+    const params = { skipTags : 1 };
```

Comment:
Since you are using skipTags as a boolean, why use '1' instead of 'true'?

Reply:
Because of type coercion and javascript being a terrible language to work in.  Try this:

```js
// as strings of numbers
'1' == true // => true
'0' == false // => false

// as strings themselves
'true' == true // => false
'false' == false // =>true
```

Basically, even if the number are passed around as strings, they true/false properties are retained.  If "true" and "false" are passed around as strings, it changes the `if (bool) { }` behavior.

... that being said, it's much more readable to use booleans, so I'll change it and figure out where we have buggy code instead.